Feat/camera follows player and rotates

Rinder5 opened the pull request:
resolves #8
resolves #9

please ignore the giant block of comment which was testing grounds for
rotating camera.

Rinder5 committed:
feat: camera follows player object

please ignore the giant block of comment which was testing grounds for
rotating camera.

nicholaschuayunzhi commented on Assets/Scripts/CameraController.cs:
```diff
@@ -0,0 +1,62 @@
+    private Vector3 _offset;
+    private float _yValue;
+
+    public float Speed = 0.25f;
```
style: could you group public and private variables together ? 😊

nicholaschuayunzhi commented on Assets/Scripts/CameraController.cs:
```diff
@@ -0,0 +1,62 @@
+        {
+            RotateCamera();
+        }*/
+    }
```
chore: could you remove the commented blocks for this PR?

nicholaschuayunzhi commented on Assets/Scripts/CameraController.cs:
```diff
@@ -0,0 +1,62 @@
+        newPosition.y = _yValue;
+        transform.position = newPosition;
+    }
+}
```
style: **let's** end a new line at the end of file!

nicholaschuayunzhi reviewed (changes requested):
Hi @Rinder5 the math looks sound. 💯 Just need to clean up the pr. Also, we need to decide whether to follow the player, or just fixate on a single point on the stage!

Rinder5 committed:
feat: camera can be rotated

need to add an input "CameraControl". negative rotates the camera
clockwise, positive rotates the camera counterclockwise.
Currently very buggy when the rotation goes full circle.

Rinder5 committed:
chore: clean up code
